src/Tickets/Commerce/Gateways/Stripe/Settings.php

```diff
@@ -147,21 +147,47 @@ private function set_connection_status() {
 	 * @inheritDoc
 	 */
 	public function get_settings() {
+		$currency_code = Currency::get_currency_code();
+		$currency_name = Currency::get_currency_name( $currency_code );
+		$plus_link     = sprintf(
+			'<a href="https://evnt.is/19zl" target="_blank" rel="noopener noreferrer">%s</a>',
+			esc_html__( 'Learn more', 'event-tickets' )
+		);
+
+		$payment_methods_tooltip = sprintf(
+			// Translators: %1$s: Opening `<strong>` tag. %2$s: The currency name. %3$s: Closing `</strong>` tag. %4$s: Opening `<a>` tag for Stripe link. %5$s: Closing `</a>` tag.
+			__( 'Payment methods available for %1$s%2$s%3$s.<br /><br /> The payment methods listed here are dependent on the currency selected for Tickets Commerce and the currency each payment method support. You can review the payment methods and their availablity for each currency on %4$sStripe\'s documentation%5$s.<br /><br />', 'event-tickets' ),
+			'<strong>',
+			$currency_name,
+			'</strong>',
+			'<a href="https://stripe.com/docs/payments/payment-methods/integration-options" target="_blank" rel="noopener noreferrer">',
+			'</a>'
+		);
+
+		$stripe_message = sprintf(
+			// Translators: %1$s: The Event Tickets Plus link.
+			esc_html__( 'You are using the free Stripe payment gateway integration. This includes an additional 2%% fee for processing ticket sales. This fee is removed by activating Event Tickets Plus. %1$s.', 'event-tickets' ),
+			$plus_link,
+		);
 		$settings = [
-			'tickets-commerce-stripe-commerce-configure'             => [
+			'tickets-commerce-stripe-commerce-description' => [
+				'type' => 'html',
+				'html' => '<div class="tec-tickets__admin-settings-tickets-commerce-description">' . $stripe_message . '</div>',
+			],
```

Comment:
I'm gonna hook to the settings from ET+ and remove this if there's a valid license in place.